src/requests/createFrontendProject.ts

```diff
@@ -36,12 +38,6 @@ export async function createFrontendProject(
     });
 
     if (response.data.status === "error") {
-        throw new Error(response.data.message);
-    }
-
-    if (response.data?.error?.message) {
         throw new Error(response.data.error.message);
     }
-
-    return response.data;
```

Comment:
why no return here? i would like to return the status

Reply:
This function has 2 outcomes:
- returns void which means the operation was successful
- throws an error which should be catched (on error)

I removed the return type because every time it returns, the operation was successful